storage/watcher_test.go

```diff
@@ -28,35 +31,52 @@ func TestWatcherWatchID(t *testing.T) {
 	idm := make(map[int64]struct{})
 
 	// synced watchings
-	for i := 0; i < 10; i++ {
-		id, cancel := w.Watch([]byte("foo"), false, 0)
+	for i := 0; i < testN/2; i++ {
```

Comment:
let's keep the previous test. we should try out best to make sure one independent change set in one pr.